pkgs/by-name/qt/qtalarm/package.nix

```diff
@@ -0,0 +1,67 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  libsForQt5,
+  qt5,
+  makeDesktopItem,
+  nix-update-script,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "qtalarm";
+  version = "2.5.1";
+
+  src = fetchFromGitHub {
+    owner = "CountMurphy";
+    repo = "QTalarm";
+    rev = "refs/tags/${finalAttrs.version}";
+    hash = "sha256-87w5YFQ9olLnCfPF04jOnIMn1NtE2M2n5WZX4e69UGU=";
+  };
+
+  buildInputs = [
+    libsForQt5.qtbase
+    libsForQt5.qtmultimedia
+  ];
+
+  installPhase = ''
```

Comment:
if stdenv.isDarwin install -Dm755 qtalarm.app -t $out/Applications else ...

Reply:
qtalarm.app is a folder, so use mv instead of install.